Lib/fontbakery/profiles/adobefonts.py

```diff
@@ -76,13 +76,20 @@ def _quick_and_dirty_glyph_is_empty(font, glyph_name):
 
 
 @check(
-    id = 'com.adobe.fonts/check/find_empty_letters',
-    rationale = """
-        Font language, script, and character set tagging approaches typically have an underlying assumption that letters (i.e. characters with Unicode general category 'Ll', 'Lm', 'Lo', 'Lt', or 'Lu', which includes CJK ideographs and Hangul syllables) with entries in the 'cmap' table have glyphs with ink (with a few exceptions, notably the Hangul "filler" characters).
-
-        This check is intended to identify fonts in which such letters have been mapped to empty glyphs (typically done as a form of subsetting). Letters with empty glyphs should have their entries removed from the 'cmap' table, even if the empty glyphs are left in place (e.g. for CID consistency).
+    id="com.adobe.fonts/check/find_empty_letters",
+    rationale="""
+        Font language, script, and character set tagging approaches typically have an
+        underlying assumption that letters (i.e. characters with Unicode general
+        category 'Ll', 'Lm', 'Lo', 'Lt', or 'Lu', which includes CJK ideographs and
+        Hangul syllables) with entries in the 'cmap' table have glyphs with ink (with
+        a few exceptions, notably the Hangul "filler" characters).
+
+        This check is intended to identify fonts in which such letters have been mapped
+        to empty glyphs (typically done as a form of subsetting). Letters with empty
+        glyphs should have their entries removed from the 'cmap' table, even if the
+        empty glyphs are left in place (e.g. for CID consistency).
     """,
```

Comment:
Since FontBakery supports several different reporters, we do not break lines in rationale text, as FB itself will perform line-breaking, if needed, depending on the target reporter.

One example of bad output caused by this change is what we get, as a result, in the text terminal:
![Screenshot from 2022-04-14 14-37-25](https://user-images.githubusercontent.com/213676/163443514-aff47a83-3e12-4a51-87d1-387e4cc3acdc.png)

Without this change, this is the much better output we originally had:
![Screenshot from 2022-04-14 14-39-12](https://user-images.githubusercontent.com/213676/163443794-f75c1179-95ac-48ee-842d-cf146513eaac.png)

Reply:
> as FB itself will perform line-breaking, if needed, depending on the target reporter.

Understood, thanks for the feedback.
Those lines are also important for the developers, and right now the experience is very poor.
I'm going to look into tweaking the reporter code.

Reply:
That would be great!

Reply:
Since this is directly related to the code-style discussion at #3396 and #3697, I'd like to share how long-lines show up on my text editor:

![Screenshot from 2022-04-14 15-20-08](https://user-images.githubusercontent.com/213676/163453004-a170fff8-2bc4-4287-8727-0a6674e54398.png)

That does not look to me like "very poor experience", so I'd appreciate if you could share in more detail what, in your opinion, constitutes a poor experience regarding this case of long lines in rationale text.

Reply:
Certainly. Below is how it shows for me, without & with line-wrapping turned on.
You have it turned on. I don't because it has the potential of masking code bugs, especially in a language that relies on correct indentation.

<img width="2103" alt="Screen Shot 2022-04-14 at 11 29 34" src="https://user-images.githubusercontent.com/2119742/163454395-1f771a3f-e545-497c-8f4d-4782edd370ea.png">
<img width="2103" alt="Screen Shot 2022-04-14 at 11 30 04" src="https://user-images.githubusercontent.com/2119742/163454427-945efdc6-2c9e-4f23-93ba-1f3e8afc90a4.png">

Reply:
ok, cool. I see your point.

Reply:
I've updated the method that processes the `rationale` string. LMK if it breaks anything.